src/MTGCore.ClientApp/ClientApp/src/App.vue

```diff
@@ -29,14 +29,24 @@
                   </div>
                   <small id="emailHelp" class="form-text text-muted">Lorem Ipsum Dolar set ammet</small>
                 </div>
-                <ResultsTable v-bind:post="post" />
+                <ResultsTable v-on:addCardToDeck="addToDeck" v-bind:post="post" />
```

Comment:
this listens for a "addCardToDeck" event which is emmited in a child component. 

best practice for vuejs is passing data down to child via props and from child to parent using events.

Reply:
Is the addCardToDeck just a custom event name? I understand addToDeck is the method below.

Reply:
i think so...
https://vuejs.org/v2/guide/components-custom-events.html#Event-Names
it says to use kebab case as well so i wiill change